web/style.css

```
@@ -498,41 +551,73 @@ input[type="search"]::placeholder { font-size: .82em; }
-  display: flex; align-items: center; gap: 10px; width: 100%;
-  font: inherit; text-align: left; padding: 6px 8px;
-  border: 1px solid var(--line-2); border-radius: 3px;
-  background: var(--panel-2); color: var(--ink-2);
-}
+/* Wikipedia の W。字体は書体が持つ (detail.mjs の WIKIPEDIA_ICON)。並べた
+   書体はどれも同じオールドスタイルのセリフで、先頭から入っているものが使われ
+   る。おにぎりの番号 (.shield text) と同じく、寸法は svg 側の viewBox・
+   textLength と目で合わせてある。 */
+.detail-wiki { text-decoration: none; }
+.detail-wiki text {
+  fill: currentColor; font-size: 22px; font-weight: 400;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)